lib/vehicle_factory.rb

```diff
@@ -0,0 +1,14 @@
+class VehicleFactory
+
+  def create_vehicles(database)
+    database.map do |data|
+      vehicle = Vehicle.new({
+        vin: data[:vin_1_10], 
+        year: data[:model_year], 
+        make: data[:make], 
+        model: data[:model], 
+        engine: :ev
+      })
+    end
```

Comment:
This works! Again though, would want an attribute to store these vehicle instances in so we don't have to map over everytime we want to access this data.